cuda_core/tests/example_tests/test_basic_examples.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021-2024 NVIDIA Corporation.  All rights reserved.
+#
+# Please refer to the NVIDIA end user license agreement (EULA) associated
+# with this source code for terms and conditions that govern your use of
+# this software. Any use, reproduction, disclosure, or distribution of
+# this software and related documentation outside the terms of the EULA
+# is strictly prohibited.
+
+# If we have subcategories of examples in the future, this file can be split along those lines
+
+from utils import run_example

```

Comment:
Could you add an empty `tests/example_tests/__init__.py` and do a relative import here? It should make it nicer to run tests everywhere.
```suggestion
from .utils import run_example
```